xbee_remote_at_block.py

```diff
@@ -35,7 +37,12 @@ def process_signals(self, signals):
     def _remote_at(self, command, parameter, dest_addr):
         command = command.encode('ascii')
         parameter = binascii.unhexlify(parameter)
-        dest_addr = binascii.unhexlify(dest_addr) if dest_addr else b'\xFF\xFF'
+        if dest_addr:
```

Comment:
For those whole section I would maybe do something like `dest_addr = binascii.unhexlify(dest_addr) if dest_addr else None`. And then in `send` do `dest_addr or b'\xFF\xFF'` and `dest_addr or b'\x00\x00\x00\x00\x00\x00\xFF\xFF'`